tests/typecheck/db/test_connection.yml

```diff
@@ -0,0 +1,13 @@
+-   case: raw_default_connection
```

Comment:
Should we test something else, what do you think? SInce there are a lot of changes 🤔